docker-compose.comms.yml

```diff
@@ -16,145 +17,91 @@ x-nats-nodes-healthy: &nats-nodes-healthy
-            "id": "discovery-node::1",
-            "spId": "5",
-            "endpoint": "http://audius-protocol-comms-discovery-1",
-            "delegateOwnerWallet": "0x123d0813710D55A9C38A2D7BC502Ff00A1a0279e",
-            "owner": { "id": "0x339511506f7BfB5f5d7042b450B9D450626dbB91" },
-            "type": { "id": "discovery-node" }
-          },
-          {
-            "id": "discovery-node::2",
-            "spId": "6",
-            "endpoint": "http://audius-protocol-comms-discovery-2",
-            "delegateOwnerWallet": "0x9262c9A1b172fF2A9C027B55862389d00b68A26a",
-            "owner": { "id": "0x11327A21bc4dE71a1274D7C1e2c94D50AdeeBB88" },
-            "type": { "id": "discovery-node" }
-          },
-          {
-            "id": "discovery-node::3",
-            "spId": "7",
-            "endpoint": "http://audius-protocol-comms-discovery-3",
-            "delegateOwnerWallet": "0xBc2550Ae74fbbDF4eC30A8E65dd845a5a8510eb4",
-            "owner": { "id": "0x339511506f7BfB5f5d7042b450B9D450626dbB91" },
-            "type": { "id": "discovery-node" }
-          }
-        ]'
+  AUDIUS_DEV_ONLY_REGISTERED_NODES: ' [ { "id": "content-node::1", "spId": "1", "endpoint": "http://audius-protocol-storage-1", "delegateOwnerWallet": "0x1c185053c2259f72fd023ED89B9b3EBbD841DA0F", "owner": { "id": "0x339511506f7BfB5f5d7042b450B9D450626dbB91" }, "type": { "id": "content-node" } }, { "id": "content-node::2", "spId": "2", "endpoint": "http://audius-protocol-storage-2", "delegateOwnerWallet": "0x90b8d2655A7C268d0fA31758A714e583AE54489D", "owner": { "id": "0x11327A21bc4dE71a1274D7C1e2c94D50AdeeBB88" }, "type": { "id": "content-node" } }, { "id": "content-node::3", "spId": "3", "endpoint": "http://audius-protocol-storage-3", "delegateOwnerWallet": "0xb7b9599EeB2FD9237C94cFf02d74368Bb2df959B", "owner": { "id": "0x339511506f7BfB5f5d7042b450B9D450626dbB91" }, "type": { "id": "content-node" } }, { "id": "content-node::4", "spId": "4", "endpoint": "http://audius-protocol-storage-4", "delegateOwnerWallet": "0xfa4f42633Cb0c72Aa35D3D1A3566abb7142c7b16", "owner": { "id": "0x11327A21bc4dE71a1274D7C1e2c94D50AdeeBB88" }, "type": { "id": "content-node" } }, { "id": "discovery-node::1", "spId": "5", "endpoint": "http://audius-protocol-comms-discovery-1", "delegateOwnerWallet": "0x123d0813710D55A9C38A2D7BC502Ff00A1a0279e", "owner": { "id": "0x339511506f7BfB5f5d7042b450B9D450626dbB91" }, "type": { "id": "discovery-node" } }, { "id": "discovery-node::2", "spId": "6", "endpoint": "http://audius-protocol-comms-discovery-2", "delegateOwnerWallet": "0x9262c9A1b172fF2A9C027B55862389d00b68A26a", "owner": { "id": "0x11327A21bc4dE71a1274D7C1e2c94D50AdeeBB88" }, "type": { "id": "discovery-node" } }, { "id": "discovery-node::3", "spId": "7", "endpoint": "http://audius-protocol-comms-discovery-3", "delegateOwnerWallet": "0xBc2550Ae74fbbDF4eC30A8E65dd845a5a8510eb4", "owner": { "id": "0x339511506f7BfB5f5d7042b450B9D450626dbB91" }, "type": { "id": "discovery-node" } } ]'
```

Comment:
aww, the format was nice

Reply:
True, I'm surprised this got auto-formatted, I'll see what I can do